smsdk/client.py

```diff
@@ -534,6 +556,14 @@ def get_lines(self, **kwargs):
         )
         return lines(self.session, base_url).get_lines(**kwargs)
 
+    one_day_relative = {
+        "time_type": "relative",
+        "relative_start": 1,
+        "relative_unit": "day",
+        "ctime_tz": "America/Los_Angeles",
+    }
```

Comment:
What does this definition exist for?

Reply:
It might be creeped in during code merging. It is not needed.